### ISSUE TITLE
Access to https://messages.google.com/web is being blocked.

### ISSUE THREAD

**Issue by GitALongLilDoggy** [mozilla/contain-facebook#745]
- Facebook Container Version: 2.1.2
- Operating System + Version: macOS 10.14.6
- Firefox Version: 87.0
- Other installed Add-ons + Version + Enabled/Disabled-Status: All others user-added Add-ons disabled

Add-ons:
Name 	Type 	Version 	Enabled 	ID
Amazon.com	extension	1.3	true	amazondotcom@search.mozilla.org
Bing	extension	1.3	true	bing@search.mozilla.org
DuckDuckGo	extension	1.1	true	ddg@search.mozilla.org
eBay	extension	1.3	true	ebay@search.mozilla.org
Facebook Container	extension	2.1.2	true	@contain-facebook
Google	extension	1.1	true	google@search.mozilla.org
Wikipedia (en)	extension	1.1	true	wikipedia@search.mozilla.org
Adblock Plus - free ad blocker	extension	3.10.2	false	{d10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d}
Form History Control (II)	extension	2.5.5.1	false	formhistory@yahoo.com
HTTPS Everywhere	extension	2021.1.27	false	https-everywhere@eff.org
LastPass: Free Password Manager	extension	4.67.1.4	false	support@lastpass.com
OneNote Web Clipper	extension	3.8.2	false	Clipper@OneNote.com
Open Link with New Tab	extension	1.0	false	Open-Link-with-New-Tab@sienori
Privacy Badger	extension	2021.2.2	false	jid1-MnnxcxisBPnSXQ@jetpack
Tab Mix - Links	extension	0.1.0	false	webext@tabmixplus.org
Tab Session Manager	extension	6.4.2	false	Tab-Session-Manager@sienori

### Actual behavior

Recently (sometime in the last two or three revs of Firefox), access to https://messages.google.com/web is being blocked in Firefox. Access was available prior to the recent past (not sure when access was first blocked; I don't use it every day; I first noticed the problem a couple of weeks ago). I can access the URL from:
Safari Version 14.0.3 (14610.4.3.1.7),
Brave Version 1.22.67 Chromium: 89.0.4389.90 (Official Build) (x86_64),
Chrome Version 89.0.4389.90 (Official Build) (x86_64), &
Vivaldi 3.7.2218.49 (Stable channel) (x86_64).

TLDR: If Firefox Container is enabled, https://messages.google.com/web will not load. There is no error message; just a blank page is displayed. Disabling Firefox Container or adding google.com to Facebook Container's Allowed Sites enables loading of https://messages.google.com/web.

### Expected behavior

https://messages.google.com/web loaded and all features on the page worked previous to the recent past. I'd like that function back, please. :-)

**Comment by maxxcrawford** [comment]
I'm unable to reproduce this in a clean install of Firefox/Facebook Container. 

I recommend trying a few a things to confirm this is being caused by Facebook Container. 

- Clear [site browser data](https://support.mozilla.org/en-US/kb/clear-cookies-and-site-data-firefox#w_clear-cookies-for-any-website) for *.google.com and *.facebook.com domains 
- [Turn off](https://support.mozilla.org/en-US/kb/disable-or-remove-add-ons) all other add-ons EXCEPT for Facebook Container. 
- Close/restart browser 

If you're still getting the blank screen, please do the following: 
- Go to `about:devtools-toolbox?id=%40contain-facebook&type=extension` in Firefox. (This will open up the developer console for the add-on)  
- Reload the https://messages.google.com page
- Go back to the devtools tab to see if there are any errors logged. (It will be under the console tab)

Let me know if you're still getting the blank page with these changes. Thanks!

**Comment by GitALongLilDoggy** [comment]
MaxxCrawford:

Well, dammit. I tried to get this right before I submitted a ticket...

Regarding your instructions, I tried the conservative approach first. I removed site browser data for messages.google.com & *.facebook.com. I turned off all Add-ons except Facebook Container, and then closed/restarted Firefox.

The URL in question loaded just fine. I re-enabled all other Add-ons, closed/restarted Firefox, and the URL loads/works without issue.

Sorry about that—thanks for your help.

:-)

**Comment by maxxcrawford** [comment]
@GitALongLilDoggy No problem at all! Happy it's back to working as expected. 😄
